modules/docs/mdx/6.0-UPGRADE-GUIDE.mdx

```diff
@@ -76,29 +76,31 @@ help us maintain additional codemod transforms to make future upgrades easier.
 
 ### Deprecation Types
 
-There are two types of deprecations: soft and hard.
+There is `deprecated` and `removal`.
 
-#### Soft Deprecation
+#### Deprecation
 
-A soft-deprecated component is still available with its full functionality, but it will have been
-renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
-to announce its deprecation. This warning will only be triggered on the component's initial render.
+We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) from JSDoc to code that we plan to
+deprecate in the near future. The component name will also have a strikethrough to indicate its
```

Comment:
Where does it have a strikethrough? The context of this is missing and might not be safe to assume.

Reply:
The strikethrough is the JSX Name of the component but I removed the mention of it since it's confusing.